test/file.jl

```diff
@@ -47,9 +64,12 @@ end
 
 # test links
 @unix_only @test islink(link) == true
+@unix_only @test readlink(link) == file
+
 @non_windowsxp_only begin
     @test islink(dirlink) == true
     @test isdir(dirlink) == true
+    @unix_only @test readlink(dirlink) == subdir
```

Comment:
Thanks for noticing it.

BTW, do I have to create a new pullrequest or is it fine how I did it below ` 	Removed functions OS limitations`